src/Common/tests/System/PlatformDetection.cs

```diff
@@ -18,9 +18,10 @@ public static partial class PlatformDetection
         // do it in a way that failures don't cascade.
         //
 
-        public static bool IsUap => IsWinRT || IsNetNative;
+        public static bool IsUap => IsInAppContainer || IsNetNative;
```

Comment:
>UAP (equivalent to F5 UWP, which is IsInAppContainer and not native

We can't change this for the definition of 'IsUap" which I recently added.  It *must* be true regardless of whether or not it is net native compiled.